app/views/layouts/embed.html.erb

```diff
@@ -28,6 +24,8 @@ Unless required by applicable law or agreed to in writing, software distributed
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <%= favicon_link_tag %>
     <%= stylesheet_link_tag "application", media: "all" %>
+    <%= stylesheet_pack_tag 'application' %>
+    <%= javascript_pack_tag 'application' %>
```

Comment:
I'm slightly concerned that including the whole application JS/CSS packs will make for a large download for only the embed.  I wonder if we can create a new pack that is just what is needed for the embed player.  Or maybe the size isn't that large and this isn't really a problem.

Reply:
It looks like our compiled assets come out to around 8MB, which feels like a lot of extra data for just a player? I'll take a look at creating a new pack and trimming that down.

Reply:
Thanks!  Good luck! :crossed_fingers: